experimental/trusted_shuffler/trusted_shuffler/src/lib.rs

```diff
@@ -138,34 +162,41 @@ impl TrustedShuffler {
     }
 
     // Lexicographically sorts requests and sends them to the backend using the
-    // [`TrustedShuffler::request_handler`].
+    // [`TrustedShuffler::request_handler`]. Blocks until all responses are received, except if a
+    // timeout is set, then it sends a default response.
```

Comment:
Does it mean that we can't process a new request batch until the first batch is done?

Reply:
Rephrased to clarify.